docs/source/strategies/wled.rst

```diff
@@ -6,7 +6,7 @@ Supported domains: ``light``
 
 You can use WLED strategy for light strips which are controlled by [WLED](https://github.com/Aircoookie/WLED).
 WLED calculates estimated current based on brightness levels and the microcontroller (ESP) used.
-Powercalc asks to input the voltage on which the lightstrip is running and optionally a power factor. Based on these factors the wattage is calculated.
+Powercalc asks to input the voltage on which the lightstrip is running and optionally a power factor. Based on these factors the wattage is calculated (Important notice: The brightness limiter must be turned on in WLED for this to work! Otherwise WLED will not provide an estimated current).
```

Comment:
You can also mark this as a block like this:
https://sublime-and-sphinx-guide.readthedocs.io/en/latest/notes_warnings.html
To give more emphasis to it. It's used multiple times in the docs where you see the blue, orange and green boxes.

I think you can use this:

```
.. important::
    The brightness limiter must be turned on in WLED for this to work! Otherwise WLED will not provide an estimated current
```